xml/System/RuntimeTypeHandle.xml

```diff
@@ -330,7 +330,7 @@
 ## Remarks  
  Using this operator to compare two variables of type <xref:System.RuntimeTypeHandle> causes an ambiguous overload resolution error when compiled. Use the <xref:System.RuntimeTypeHandle.Equals%2A> method instead.  
   
- The equivalent method for this operator is <xref:System.RuntimeTypeHandle.Equals%28System.RuntimeTypeHandle%2CSystem.Object%29?displayProperty=fullName>]]></format>
+ The equivalent method for this operator is <xref:System.RuntimeTypeHandle.Equals%28System.RuntimeTypeHandle%29?displayProperty=fullName>.]]></format>
```

Comment:
I've actually added system.object but hopefully with the right signature to the specific method